addons/binding/pom.xml

```diff
@@ -93,6 +93,7 @@
     <module>org.openhab.binding.synopanalyzer</module>
     <module>org.openhab.binding.systeminfo</module>
     <module>org.openhab.binding.systeminfo.test</module>
+    <module>org.openhab.binding.tado</module>
```

Comment:
You also need to add your binding to the [feature.xml](https://github.com/openhab/openhab2-addons/blob/master/features/openhab-addons/src/main/feature/feature.xml) file.

Reply:
Done